tools/oeedger8r/src/Sources.ml

```diff
@@ -81,7 +81,9 @@ let _get_param_size (ptype, decl, argstruct) =
     and count = attr_value_to_string argstruct p.ps_count in
     match (size, count) with
     | Some s, None -> s
-    | None, Some c -> sprintf "(%s * sizeof(%s))" c type_expr
+    (* TODO: Check that c actually fits in size_t. Also check for overflow,
+     * similar to oe_add_size *)
```

Comment:
Implement this in this PR rather than waiting until later.

Reply:
Actually this is a bigger change than I want to do here. Opened an issue here: #2390